mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_job_log_entries_function.sql

```diff
@@ -119,12 +120,13 @@ FROM (
                 (array_agg(entity_type) FILTER (WHERE entity_type IN (''EDIFACT'')))[1] AS source_record_entity_type,
                 array[]::varchar[] as holdings_entity_hrid
          FROM journal_records
-         WHERE journal_records.job_execution_id = ''%s'' and entity_type = ''INVOICE'' and title != ''INVOICE''
+         WHERE journal_records.job_execution_id = ''%1$s'' and entity_type = ''INVOICE'' and title != ''INVOICE''
          GROUP BY journal_records.source_id, journal_records.source_record_order, journal_records.job_execution_id,
                   entity_hrid, title, error, id
      ) AS records_actions
-ORDER BY %I %s
-LIMIT %s OFFSET %s;',
-                              jobExecutionId, jobExecutionId, jobExecutionId, jobExecutionId, v_sortingField, sortingDir, limitVal, offsetVal);
+WHERE NOT %2$L or error = '''' IS FALSE
```

Comment:
Adding a comment here that explains this line could help.

Reply:
Done